es6-persistence/build.gradle

```diff
@@ -11,6 +11,8 @@
  *  specific language governing permissions and limitations under the License.
  */
 
+ext['elasticsearch.version'] = revElasticSearch6
```

Comment:
This is not necessary, since the root `build.gradle` already includes it.

Reply:
I think it can be a preferable way to avoid problems with transitive dependencies in es-persistence modules. For example, change of this param's value from revElasticSearch6 to revElasticSearch7 in the root build.gradle shouldn't lead to transitive dependencies problems with the es6-persistence module and on the contrary.

Reply:
Valid point. The way we support ES is that we have one default version and one or more optional versions. For v3.x, the default is `6.x`. Spring Boot 2.3.x's default is `7.x`. That is why, we had to set version to ES6 in root, `build.gradle`. In all of the modules es version should be `6.x` except the one introduced in this PR `es7-persistence`. When we switch to `7.x` as the default, we will change the root `build.gradle`  to `7.x` and shade `6.x` libraries.
```
ext['elasticsearch.version'] = revElasticSearch6
```

Reply:
removed from es6 module